modular_skyrat/modules/modular_reagents/code/modifiers/reagent.dm

```diff
@@ -0,0 +1,9 @@
+//*
```

Comment:
This isn't how we do multiline comments, look up the Autodoc Comment Block extension on VSC, I can link it to you on Discord if I can find it. This is really funky and hurts my soul a little.